charts/datahub/subcharts/datahub-gms/templates/config-jmx-exporter.yaml

```diff
@@ -10,6 +10,18 @@ data:
     hostPort: localhost:{{ .Values.env.JMXPORT }}
     lowercaseOutputName: {{ .Values.exporters.jmx.config.lowercaseOutputName }}
     lowercaseOutputLabelNames: {{ .Values.exporters.jmx.config.lowercaseOutputLabelNames }}
+    {{- if .Values.exporters.jmx.config.whitelistObjectNames }}
```

Comment:
if statement is redundant here. with will only get triggered if the value exists! same for all other parts hre

Reply:
> Sorry about the delay @claudio-benfatto minor comment. You also need to up chart version!

np! Thanks for the review, I made the changes you suggested ;)